src/components/ui/MdxInfoBlock.tsx

```diff
@@ -0,0 +1,16 @@
+import { ReactElement } from "react-markdown/lib/react-markdown";
```

Comment:
We can directly use ReactElement from react here

```suggestion
import { ReactElement } from "react";
```